tests/molecule.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+failed_roles=()
+
+for roledir in roles/*/molecule; do
+  role_path=$(dirname $roledir)
+  role_name=$(basename $role_path)
+  pushd ${role_path}
+  molecule test --all || failed_roles+=($role_name)
+  popd
```

Comment:
You can try to create groups for easier debugging: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines
```suggestion
  echo "::group::Ansible role $role_name"
  pushd ${role_path}
  molecule test --all || failed_roles+=($role_name)
  popd
  echo "::endgroup::"
```

Reply:
Oh, that's nifty!

Reply:
Molecule already does grouping, if it detects GHA, and nested groups seem to be broken: https://github.com/actions/runner/issues/802

I'll play with that idea in a follow-up.